enterprise/backend/src/metabase_enterprise/serialization/cmd.clj

```diff
@@ -47,12 +48,13 @@
     (throw (ex-info "You cannot `import` into an empty database. Please set up Metabase normally, then retry." {})))
   (when token-check?
     (check-premium-token!))
-  ; TODO This should be restored, but there's no manifest or other meta file written by v2 dumps.
```

Comment:
probably better to just fix these and give them two semicolons instead of reindenting them